Source/ChocolateyGui/Services/SettingsSourceService.cs

```diff
@@ -63,13 +54,7 @@ public void RemoveSource(SourceViewModel viewModel)
             }
 
             Settings.Default.sources.Remove(string.Format(CultureInfo.CurrentCulture, "{0}|{1}", viewModel.Name, viewModel.Url));
-            var sourcesChangedEvent = this.SourcesChanged;
-            if (sourcesChangedEvent != null)
-            {
-                sourcesChangedEvent(
-                    this,
-                    new SourcesChangedEventArgs(new List<SourceViewModel>(), new List<SourceViewModel> { viewModel }));
-            }
+            this.RaiseSourceRemovedEvent(viewModel);
```

Comment:
In all honesty, I am not sure.  I am hesitant to do any work on the Settings.Default.soures, as I see this as essentially legacy code, that will only be used by people that are still using ChocolateyGUI in combination with the PowerShell version of Chocolatey.  I think we should encourage people to upgrade to the latest Chocolatey, and eventually we can start removing some of the legacy code.

I would be happy to accept a Pull Request to make changes to the Settings.Default.sources, but I don't think I will be making any changes in this area.